libcudacxx/include/cuda/std/detail/libcxx/include/__functional_base

```diff
@@ -35,6 +35,7 @@
 #include <cuda/std/detail/libcxx/include/exception>
 #include <cuda/std/detail/libcxx/include/new>
 
+// comes last
 #include <cuda/std/detail/libcxx/include/__pragma_push>
```

Comment:
Suggestion: I found a couple of these, so we may want to aim for a systematic solution. Maybe, this include could be put last using [clang-format's `IncludeCategories`](https://clang.llvm.org/docs/ClangFormatStyleOptions.html#includecategories).

Reply:
You are right, fixed